src/ManifoldMuseum.jl

```diff
@@ -332,14 +331,13 @@ export Metric,
     LorentzMetric,
     EuclideanMetric,
     MetricManifold,
-    IsRiemannianManifold,
-    isriemannianmanifold,
+    HasMetric,
```

Comment:
One could also use `IsMetricManifold` which is a little more consistent with the `IsDecoratorManifold`, but I would be fine with either naming.

Reply:
I removed `IsRiemannianManifold` because I don't have a great use for it right now. It's easy to dispatch on whether a `MetricManifold` is using a Riemannian metric, but none of the functionality I'm implementing so far is unique to Riemannian manifolds, so this seems overkill.